<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="40060" started-at="2017-05-10T15:54:32Z" finished-at="2017-05-10T15:55:12Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="40060" started-at="2017-05-10T15:54:32Z" finished-at="2017-05-10T15:55:12Z">
      <class name="tests.TestAutomation">
        <test-method status="FAIL" signature="beforeTest()[pri:0, instance:tests.TestAutomation@259847e5]" name="beforeTest" is-config="true" duration-ms="0" started-at="2017-05-10T21:24:32Z" finished-at="2017-05-10T21:24:32Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Failed : HTTP error code : 401]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Failed : HTTP error code : 401
	at utils.ZAPI.postOrPut(ZAPI.java:147)
	at utils.ZAPI.postOrPut(ZAPI.java:136)
	at utils.ZAPI.postOrPut(ZAPI.java:136)
	at utils.ZAPI.postOrPut(ZAPI.java:136)
	at core.JIRA_Reporter.cloneTestCycle(JIRA_Reporter.java:72)
	at tests.TestAutomation.beforeTest(TestAutomation.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="SKIP" signature="sampleTest()[pri:0, instance:tests.TestAutomation@259847e5]" name="sampleTest" duration-ms="2" started-at="2017-05-10T21:25:12Z" finished-at="2017-05-10T21:25:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sampleTest -->
        <test-method status="SKIP" signature="afterTest()[pri:0, instance:tests.TestAutomation@259847e5]" name="afterTest" is-config="true" duration-ms="0" started-at="2017-05-10T21:25:12Z" finished-at="2017-05-10T21:25:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- tests.TestAutomation -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
